addons/website_event_booth_sale/models/sale_order.py

```diff
@@ -42,9 +42,49 @@ def _website_product_id_change(self, order_id, product_id, qty=0, **kwargs):
                     event_booth_registrations_command = new_registrations_commands
                 values['event_booth_registration_ids'] = event_booth_registrations_command
 
+            discount = 0
+            order = self.env['sale.order'].sudo().browse(order_id)
+            booth_currency = booths.product_id.currency_id
+            pricelist_currency = order.pricelist_id.currency_id
+            price_reduce = sum(booth.booth_category_id.price_reduce for booth in booths)
+            if order.pricelist_id and order.partner_id:
+                order_line = order._cart_find_product_line(booths.product_id.id)
+                if order_line:
+                    price_reduce = self.env['account.tax']._fix_tax_included_price_company(price_reduce, booths.product_id.taxes_id, order_line[0].tax_id, self.company_id)
```

Comment:
Is it necessary to duplicate the tax logic on price and price_reduce ?
Can't we simply apply it after the `if order.pricelist_id.discount_policy == 'without_discount' ... else ...`, directly on price_unit ?